### ISSUE TITLE
Map cmd-click to ctrl-click for Apple platforms

### ISSUE THREAD

**Issue by MutatingFunc** [GardensOfKadesh/Homeworld#25]
On my iPad, ctrl-click will perform a right click, preventing me from issuing attack moves / orders. Remapping modifiers doesn’t seem to prevent this behaviour. This is a holdover from Mac, so is probably an issue there too.

Mapping command-click to ctrl-click should solve the issue.

**Comment by rbrune** [comment]
Thank you for this and the other report. I think you're the first to try it on an iPad.
Just to understand your setup. you used an iPad with an apple keyboard and mice attached?

**Comment by MutatingFunc** [comment]
Yep!I can’t remember how much I said in the report but yeah, ctrl-click to right click is standard on Apple platforms.As a Playgrounds for iPad app developer in my free time, I tend to be the first to try a lot on iPad. 😄I made a tiny Playgrounds app to host your webpage fullscreen too, in the hopes I can play properly sometime! Making it a PWA could be awesome too.Give me a shout if I can test anything at all!JamesOn 4 Oct 2024, at 22:11, Rafael Brune ***@***.***> wrote:﻿
Thank you for this and the other report. I think you're the first to try it on an iPad.
Just to understand your setup. you used an iPad with an apple keyboard and mice attached?

**Comment by MutatingFunc** [comment]
Hold up, it’s an a bundle format so I bet I need to zip it. Though you might not even have an iPad/Mac to open it. Well, doesn’t matter too much, feel free to ignore. 😅On 4 Oct 2024, at 22:36, James Froggatt ***@***.***> wrote:﻿Yep!I can’t remember how much I said in the report but yeah, ctrl-click to right click is standard on Apple platforms.As a Playgrounds for iPad app developer in my free time, I tend to be the first to try a lot on iPad. 😄I made a tiny Playgrounds app to host your webpage fullscreen too, in the hopes I can play properly sometime! Making it a PWA could be awesome too.<Homeworld>Give me a shout if I can test anything at all!JamesOn 4 Oct 2024, at 22:11, Rafael Brune ***@***.***> wrote:﻿
Thank you for this and the other report. I think you're the first to try it on an iPad.
Just to understand your setup. you used an iPad with an apple keyboard and mice attached?
